webapp/components/NavbarLoginButton/NavbarLoginButton.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <NuxtLink v-if="loggedOut" class="btn btn-outline-success" role="button" to="/login">
+    Login
+  </NuxtLink>
+  <button v-else class="btn btn-outline-danger" @click="logout">
+    Logout
+  </button>
+</template>
```

Comment:
⭐ ⭐ **For a menu component which shows a login or logout button** and its software tests.